scss/_patterns_chip.scss

```diff
@@ -10,6 +10,7 @@
     align-items: baseline;
     border-radius: 1rem;
     display: inline-flex;
+    gap: $sph--small;
```

Comment:
I think we shouldn't introduce the gap here, it affects not only a badge, but other elements inside the chip, making it have more space than intended.

You can see the difference when looking at Percy or https://vanilla-framework-4497.demos.haus/docs/examples/patterns/chip/variants